utilities/trace_analyzer_test.cc

```diff
@@ -0,0 +1,708 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+//
+// Copyright (c) 2012 The LevelDB Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file. See the AUTHORS file for names of contributors.
+
+#ifndef ROCKSDB_LITE
+#ifndef GFLAGS
+#include <cstdio>
+int main() {
+    fprintf(stderr, "Please install gflags to run trace_analyzer test\n");
+      return 1;
+}
+#else
+
+#include <stdint.h>
+#include <sstream>
+#include <thread>
+#include <chrono>
+
+#include <rocksdb/trace_analyzer_tool.h>
+#include "db/db_test_util.h"
+#include "rocksdb/db.h"
+#include "rocksdb/env.h"
+#include "rocksdb/status.h"
+#include "rocksdb/trace_reader_writer.h"
+#include "util/testharness.h"
+#include "util/testutil.h"
+#include "util/trace_replay.h"
+#include "utilities/trace_analyzer_tool_imp.h"
+
+namespace rocksdb {
+namespace {
+static const int kMaxArgCount = 100;
+static const size_t kArgBufferSize = 100000;
+
+bool ReadOneLine(std::istringstream* iss, SequentialFile* seq_file,
+                 std::string* output, bool* has_data, Status* result) {
```

Comment:
Is it the same as the one in options/options_parser.cc? If it is, reuse that one instead of duplicate the code.